index.html

```diff
@@ -37,6 +32,119 @@ <h3>I’am  web a developer.</h3>
             <li><img src="images/medium.png" alt="social-icon" /></li>
       </ul>   
     </section>
+       <section class="my-works1">
+
+
+        <h1>My Recent works</h1>
+
+     
+        <h3>Multi-Post Stories</h3>
+        <a class="img1"><img src="images/placeholder.png" alt="hamburger Icon" /></a>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        
+        
+          <ul class="lenguajes">
+            <li>css</li>
+            <li>html</li>
+            <li>boostrap</li>
+            <li>ruby</li>
+           
+          </ul>
+          
+          
+        <button type="submit">See Project</button>
+
+        </section>
+
+        <section class="my-works">
+        <div class="center">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="lenguajes">
+            <li>css</li>
+            <li>html</li>
+            <li>boostrap</li>
+            <li>ruby</li>
+           
+          </ul>
+          
+          <button type="submit">See Project</button>
+        </div>
+        
+      </section>
```

Comment:
- [ ] [Optional] For a semantic structure of your code here you could use an `article` tag, in that way you will have a `section` filled with `articles`